jwst/datamodels/schemas/resamp.schema.yaml

```diff
@@ -0,0 +1,50 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/fits-schema/fits-schema"
+id: "http://stsci.edu/schemas/jwst_datamodel/resamp.schema"
+allOf:
+- $ref: core.schema
+- $ref: photometry.schema
+- $ref: wcsinfo.schema
+- $ref: slitmeta.schema
+- type: object
+  properties:
+    meta:
+      type: object
+      properties:
+        bunit_data:
+          title: physical units of the array values
+          type: string
+          fits_hdu: SCI
+          fits_keyword: BUNIT
+        pointings:
```

Comment:
It looks to me like 2 or 3 of these keywords are also used in `resample_spec` and hence presumably they get populated in what was the `MultiProductModel` (now `MultiResampModel`), so I guess we need these 3 keywords defined in `multiresamp.schema`. Or perhaps we can again consolidate things into one place and put all drizzle/resample-related keyword definitions in a sub-schema, like "resampmeta.schema", and include that sub-schema in both `resamp.schema` and `multiresamp.schema`.